src/web/app/components/question-types/question-edit-details-form/num-scale-question-edit-details-form.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <div class="col-sm-4" ngbTooltip="Minimum acceptable response value">
     Minimum value:
-    <input type="number" class="form-control" [ngModel]="model.minScale" (ngModelChange)="triggerModelChange('minScale', $event)" [disabled]="!isEditable" min="1" [step]="model.step">
+    <input type="number" class="form-control" [ngModel]="model.minScale" (ngModelChange)="triggerModelChange('minScale', $event === null ? $event : Math.ceil($event))" [disabled]="!isEditable" min="0" [step]="1">
```

Comment:
I don't think there's a minimum amount for the minimum value, so maybe you can drop the restriction entirely.